README.md

```diff
@@ -1,169 +1,169 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Tech-stack</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="">Rspec</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **register and log in, so that the data is private to them.**
+- **Categorize purchases to see exact amounts spent on groceries, eating out, shopping, bills etc.**
+- **Set custom budgets for each category and track over/under spending**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo & Presentation <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-app-dr4e.onrender.com)
+- [Video presentation](https://drive.google.com/file/d/12L3Cbr-AQqXLpPbJ8xi-hcw6PMuangFW/view?usp=sharing)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/izebiafe/Budget-app.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  bundle install
+  rails assets:precompile
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec
+```
+
+### Deployment
+
+You can deploy this project using your own deployment service like `Render`.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Afimia Fidelis**
+
+- GitHub: [@Izebiafe](https://github.com/Izebiafe)
+- LinkedIn: [ticoniq](https://linkedin.com/in/Izebiafe)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Upload icons as files**
+- [ ] **Add location attribute to be saved for each purchase**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project consider adding a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank Microverse.
+- Original design idea by: Gregoire Vella on Behance.

```

Comment:
**OPTIONAL** - You did well acknowledging the authors of the original design, but it'd be better if you added a link leading to the design portfolio so future hiring managers can see the design and tried to replicate, please fix this 👍 👍